### PR TITLE
osa region added

### PR DESCRIPTION
**Pull request by Madhan-SWE**
**What type of PR is this?**
/kind bug

**What this PR does / why we need it**:
Added `osa` region in getRegion method.
**Which issue(s) this PR fixes**:

Fixes #

**Special notes for your reviewer**:

**Release note**:
```
none
```

### CODE REVIEW

File: pkg/cloud/powervs.go [hunk 1]
```diff
@@ -409,6 +409,8 @@ func getRegion(zone string) (region string, err error) {
 		region = "syd"
 	case strings.HasPrefix(zone, "mon"):
 		region = "mon"
+	case strings.HasPrefix(zone, "osa"):
```
Comment (mkumatag) [inline pkg/cloud/powervs.go:412]: @Prajyot-Parab see is we need to update this code any more regions

**Review by mkumatag — CHANGES_REQUESTED** [review]

File: pkg/cloud/powervs.go [hunk 2]
```diff
@@ -409,6 +409,8 @@ func getRegion(zone string) (region string, err error) {
...
+		region = "osa"
```
Comment (Prajyot-Parab) [inline pkg/cloud/powervs.go:413]:
```suggestion
	case strings.HasPrefix(zone, "osa"):
		region = "osa"
	case strings.HasPrefix(zone, "dal"):
		region = "dal"
	case strings.HasPrefix(zone, "sao"):
		region = "sao"
	case strings.HasPrefix(zone, "tok"):
		region = "tok"
```

File: pkg/cloud/powervs.go (hunk 1 above)
Comment (Madhan-SWE) [reply pkg/cloud/powervs.go:412]: missed regions added now.

**Pull request merged**